src/patterns/step-by-step-navigation/index.md

```diff
@@ -168,7 +168,7 @@ Visit the UK on a Standard Visitor visa
 **Department for Transport**<br>
 Get a Blue Badge
 
-See a full list of [live services using step by step navigation](https://live-stuff.herokuapp.com/step-by-steps).
+See a full list of [live services using step by step navigation](https://govuk-formats.herokuapp.com/).
```

Comment:
**question** @davidc-gds Should we link directly to the page with the list rather than the homepage of the site? Looks like there are many other pieces of content on that site.

```suggestion
See a full list of [live services using step by step navigation](https://govuk-formats.herokuapp.com/document-types/step-by-step-nav).
```

Reply:
Hey @romaricpascal yeah that's likely the thing to do -- thanks for catching that!